### ISSUE TITLE
Mixin to place element and set context at the same time

### ISSUE THREAD

**Issue by name-k** [kizu/bemto#75]
It would be nice to have a mixin (`+be` for example) to set a context and an element at the same time
```
+b.box
  +be.list
    +e.item
```

```
div.box
  div.box__list
    div.box__list__item
```

**Comment by name-k** [comment]
May be I'm missing something. Will appreciate any tips and guidance.

**Comment by kizu** [comment]
There is a setting and a syntax for this right now, see https://github.com/kizu/bemto#setting-for-allowing-nested-elements

``` jade
- set_bemto_settings({ flat_elements: false })

+b.box
  +e.list__
    +e.item
```

**Comment by name-k** [comment]
Actually it acts not like expected, it affects elements out of scope
```
- set_bemto_settings({ flat_elements: false })

+b.box
  +e.list__
    +e.item
  +e.another-list
```
will render to
```
.box
  .box__list
    .box__list__item
  .box__list__another-list
```

**Comment by kizu** [comment]
Which version of bemto do you use? I think this issue was fixed in v1.0.2 (via #70), as it does not reproduce for me now.

**Comment by name-k** [comment]
Yes, you are right. Thanks!
